packages/workspace/src/core/affected-project-graph/affected-project-graph.ts

```diff
@@ -10,23 +16,35 @@ import { getImplicitlyTouchedProjectsByJsonChanges } from './locators/implicit-j
 import { AffectedProjectGraphContext } from './affected-project-graph-models';
 import { normalizeNxJson } from '../normalize-nx-json';
 
+export interface TouchedProjectLocator {
```

Comment:
Affected-project-graph models might be a better place to have it

Reply:
And since you introduced it, you can update all existing locators to use the interface